src/apps/competitions/tests/test_legacy_command_replacement.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from docker.compute_worker.compute_worker import replace_legacy_metadata_command
+
+
+class LegacyConverterCommandTests(TestCase):
```

Comment:
if we want to get crazy could add cases whether we replace ingestion only during scoring, etc. etc.? Not necessary, for sure